src/Sidekick.Mock/MockAppService.cs

```diff
@@ -1,18 +1,27 @@
 using System;
 using System.Threading.Tasks;
 using Sidekick.Common;
+using Sidekick.Common.Platform.Tray;
 
 namespace Sidekick.Mock
 {
     public class MockAppService : IAppService
     {
+        private readonly ITrayProvider trayProvider;
+        public MockAppService(ITrayProvider trayProvider)
+        {
+            this.trayProvider = trayProvider;
+        }
+
         public Task OpenConfirmationNotification(string message, string title = null, Func<Task> onYes = null, Func<Task> onNo = null)
         {
             return Task.CompletedTask;
         }
 
         public Task OpenNotification(string message, string title = null)
```

Comment:
We could clean the IAppService interface. We do not need an OpenNotification on that interface anymore. We can update the code where the AppService was required to open a notification and use the new TrayProvider service now.